Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs

```diff
@@ -1135,8 +1169,17 @@ private void TransformTarget(HandleType transformType)
                     var goal = initialPositionOnGrabStart + translateVectorAlongAxis;
                     MixedRealityTransform constraintTranslate = MixedRealityTransform.NewTranslate(goal);
                     constraints.ApplyTranslationConstraints(ref constraintTranslate, true, isNear);
-                    goal = constraintTranslate.Position;
-                    Target.transform.position = smoothingActive ? Smoothing.SmoothTo(Target.transform.position, goal, translateLerpTime, Time.deltaTime) : goal;
+
+                    if (elasticsManager != null)
+                    {
+                        transformUpdated = elasticsManager.ApplyTargetTransform(constraintTranslate, TransformFlags.Move);
+                    }
+                    if (!transformUpdated.HasFlag(TransformFlags.Move))
+                    {
+                        Target.transform.position = smoothingActive ? 
+                            Smoothing.SmoothTo(Target.transform.position, constraintTranslate.Position, translateLerpTime, Time.deltaTime) : 
+                            constraintTranslate.Position;
+                    } 
```

Comment:
Maybe not necessary for this PR, but consider having the transforms calculation to be made by a separate class, according to the type of handle.

Reply:
This would also give the user more flexibility for making other kinds of crazy transformation , translating in zig zag for example :)

Reply:
yesssss! ! I was thinking about this the last cpl of days and wanted to get some feedback on how people would feel about this - I think it makes the handles easier to customize if we move the logic in there :) (though i would;ve just moved base transform changes in / not the constraint / elastic stuff... maybe a good idea to have methods for those things in the handles then as well .... i'll prototype something for another PR :) )